src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+@import "./stars.scss";
+@import "./utils/variables";
+@import "./utils/extends";
+
+.products {
+  @extend %column-direction;
+
+  align-items: center;
+  padding: 50px 0 25px;
+
+  &__row {
+    display: flex;
+    margin-bottom: $card-column-gap;
+
+    .card {
```

Comment:
Why did you write card styles inside products __row? On CSS it will be:
`.products__row .card {}`
Don't increase specificity